docs/topics/configmap-mission-intro.adoc

```diff
@@ -8,11 +9,10 @@ This mission shows you how to:
 .About Externalized Configuration
 It is important for the application configuration to be externalized and separated from its code. This allows the application configuration to change as it moves through different environments while leaving the code unchanged. This also keeps sensitive or internal information out of your code base and version control. Many languages and application servers provide environment variables to support externalizing an application's configuration.
 Microservices architectures and polyglot environments add a layer of complexity to managing an application's configuration. Applications are comprised of independent, distributed services, each potentially with its own configuration.
-This creates a maintenance challenge to keep the configuration synchronized and accessible from all services. 
+This creates a maintenance challenge to keep the configuration synchronized and accessible from all services.
 
 .About ConfigMap
```

Comment:
This should be renamed since it's not explaining what a ConfigMap is anymore. Maybe remove the sub-sections entirely and merge all the text together…  saying something to the effect of: ConfigMaps are OpenShift's solution to  externalizing configuration… Here's more info (link)… This is important because… (text from Why ConfigMap is important subsection).